PROCEDURES.md

```diff
@@ -0,0 +1,199 @@
+
+## Releasability check
+tl;dr: check that the releasability test is passing.
+### Exceptions
+Repos that don't have dependencies naturally don't need a releasability check and this step can be skipped for those. Currently, `knative.dev/hack` is the only `knative.dev` repo that does not have any dependencies.
+
+The releasability check will not work on dot releases and there is a potential for false positives in those cases.
+
+### Updating the releasability defaults
+Open a PR in the `knative-sandbox/knobots` repo to update the releasability defaults for a release and bump the version info in the source workflow [knative-releasability.yaml](https://github.com/knative-sandbox/knobots/blob/main/workflow-templates/knative-releasability.yaml) file.  Here is an example:
+
+- [Update releasability defaults for 1.5 by gab-satchi · Pull Request #202 · knative-sandbox/knobots](https://github.com/knative-sandbox/knobots/pull/202?w=1)
+
+The changes in the source workflow will be automatically propagated to the rest of Knative repos in the next round of
+workflow syncs, and it can also be triggered manually when necessary.
+
+### Verifying releasability
+An automatic repo releasability test is executed periodically and the results are posted on the corresponding Slack release channel. If the dependencies are properly aligned, the releasability test will pass.
+
+A test can be re-run by manually running the [Releasability workflow](https://github.com/knative/serving/actions?query=workflow%3AReleasability).
+
+If the releasability test reports a NO-GO on a repo where it was previously passing, probably a new PR merge introduced a dependency misalignment. Whatever the reason, if the releasability test is failing, it is necessary to start with the first step in the [Releasing a repository](#releasing-a-repository) process to get this test to a passing state.
+
+## Cut a branch
+tl;dr: cut a release branch from `main`.
+### Exceptions
+For some repositories some extra manual validation and updates need to be performed before the release branch is cut:
+
+- `knative/client`
+  - (optional) Check that [CHANGELOG.adoc](https://github.com/knative/client/blob/main/CHANGELOG.adoc) contains a section about the release, i.e. the top-level "(Unreleased)" section should be changed to point to the upcoming release version and number. It's not critical if the changelog is aligned after the release in retrospective.
+  - If the validation fails, the fix should be trivial and could be either performed by the release leads or the client team.
+
+- `knative-sandbox/kn-plugin-quickstart`
+  - Update the version numbers of Serving / Kourier / Eventing in [pkg/install/install.go](https://github.com/knative-sandbox/kn-plugin-quickstart/blob/main/pkg/install/install.go#L25-L27) so that the plugin will use the just-released versions.
```

Comment:
```suggestion
  - Update the version numbers of Serving / Kourier / Eventing in [pkg/install/install.go](https://github.com/knative-sandbox/kn-plugin-quickstart/blob/4e57ed10eaac8882b0a81b2861ddaee37bfcd6f9/pkg/install/install.go#L25-L27) so that the plugin will use the just-released versions.
```

Probably better to pin to a specific commit, as the line references might change if updates are made (this one's on me... I added this line a while back and really should've used an explicit reference :facepalm: )